lib/MetaCPAN/Server/Model/CPAN.pm

```diff
@@ -11,9 +11,9 @@ extends 'Catalyst::Model';
 with 'CatalystX::Component::Traits';
 
 has esx_model => (
-    is      => 'ro',
-    handles => ['es'],
-    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
+    is         => 'ro',
+    lazy_build => 1,
```

Comment:
This can still be `lazy => 1` with a default, but let's merge this now and we can look at it later.